src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -1726,6 +1811,8 @@ public string DeleteCounter(DocumentsOperationContext context, string documentId
             {
                 var collectionName = _documentsStorage.ExtractCollectionName(context, collection);
                 var table = GetCountersTable(context.Transaction.InnerTransaction, collectionName);
+
+                CreateCounterTombstone(context, documentKeyPrefix, counterNameSlice, collectionName);
```

Comment:
why do you need that?

Reply:
On counter delete operation (local) - we need to add a new entry to the counter tombstones table to mark this counter as pending deletion. Otherwise, we may skip writing it in the operating server.

Reply:
and we do that even if the counter doesn't exists?

Reply:
Oh, great question. I thought it should be as in TS -> but now after we did it locally, it makes sense to create a tombstone only if the counter exists. Changed.